tests/core/eth-module/test_accounts.py

```diff
@@ -156,8 +156,8 @@ def test_eth_account_recover_message(acct):
         '0x3e5bfbbf4d3e39b1a2fd816a7680c19ebebaf3a141b239934ad43cb33fcec8ce',
     )
     message = "I♥SF"
-    message_hash = defunct_hash_message(text=message)
-    from_account = acct.recoverHash(message_hash, vrs=(v, r, s))
+    message_hash = encode_defunct(text=message)
+    from_account = acct.recover_message(message_hash, vrs=(v, r, s))
```

Comment:
Since `encode_defunct` returns a (signable) message now, the variable name should probably be changed from `message_hash`. Maybe `signable_message`, `encoded_message` or `encoded` is a reasonable name now.

Another very reasonable option is to rename `message` => `message_text`, and `message_hash` => `message`. (this might by my favorite)